src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  isLoading: boolean;
+};
+
+export const Loader: React.FC<Props> = ({ isLoading }) => (
+  <div
+    data-cy="TodoLoader"
+    className={classNames('modal overlay', {
+      'is-active': isLoading,
+    })}
+  >
+    <div className="modal-background has-background-white-ter" />
+    <div className="loader" />
+  </div>
+);
```

Comment:
you have one more "loader" under the list, that work, when you delete or change todo

Reply:
You mean the one that is working when you open the page and wait for todos to load?